### PR TITLE
Submenus for moving all cards in a zone to another

### PR DESCRIPTION
**Pull request by ideocl4st**
As discussed in #584, I added submenus for moving all cards in a given zone to another zone.

I took some initiative of changing all context menus with similar effect (From exile to ~, From graveyard to ~), but I'll change it back if you feel it's unnecessary.

PS : I forgot to revert changing string for revealing the entire hand from "Reveal hand to". I'm keeping it for now since I like the change as it is more distinguishable from revealing a card, but I'll also change it back if it feels unnecessary.

### CODE REVIEW

**Comment by ZeldaZach** [comment]
Can you include pictures on the commit please? <3

**Comment by ideocl4st** [comment]
Sure, why not.
handmenu
gravemenu
rfgmenu

**Comment by ZeldaZach** [comment]
I'm a fan of this; will test in a bit

**Comment by ZeldaZach** [comment]
I fully approve and love this option!

**Comment by poixen** [comment]
Judging from the images, it looks banging. Up voted.

File: cockatrice/src/player.cpp [hunk 1]
```diff
@@ -585,18 +588,24 @@ void Player::retranslateUi()
...
-        aMoveHandToRfg->setText(tr("Move to &exile"));
-        aMoveGraveToTopLibrary->setText(tr("Move to &top of library"));
-        aMoveGraveToBottomLibrary->setText(tr("Move to &bottom of library"));
-        aMoveGraveToHand->setText(tr("Move to &hand"));
-        aMoveGraveToRfg->setText(tr("Move to &exile"));
-        aMoveRfgToTopLibrary->setText(tr("Move to &top of library"));
-        aMoveRfgToBottomLibrary->setText(tr("Move to &bottom of library"));
-        aMoveRfgToHand->setText(tr("Move to &hand"));
-        aMoveRfgToGrave->setText(tr("Move to &graveyard"));
+        moveHandMenu->setTitle(tr("&Move hand to"));
```
Comment (Daenyth) [inline cockatrice/src/player.cpp]: Minor tweak, but can you make the upper level menus "Move foo to..." and capitalize the submenu entries? After that we should be good to merge.

**Comment by ideocl4st** [comment]
@Daenyth Done.

**Comment by ideocl4st** [comment]
@Daenyth Done.

But wait, what about the 'Reveal hand to' and others? Should I put ellipsis on them too?

**Comment by ZeldaZach** [comment]
If you'd like, that would be awesome :)

**Comment by ideocl4st** [comment]
It's all about consistency. Just like running a mono red sligh deck.

newnewhandmenu
newlibrarymenu

**Comment by Daenyth** [comment]
This is just about ready but there's one thing I strongly strongly dislike -- in every menu except Hand, you've made `^M` the key for "move to", but in the hand `^M` is mulligan. This seems to me like it's going to set users up to hit mulligan by accident.

I really don't want to change mulligan, so I'd like to change the move menu keys to something else.

**Comment by ZeldaZach** [comment]
I don't think I understand; I don't see the shortcut `Shift + M` altered anywhere

**Comment by poixen** [comment]
2 things. I think open deck in deck editor should have its own separator and be at the bottom. The same with take mulligan.

**Comment by Daenyth** [comment]
Wait, derp, you're right - you don't need to hit a modifier key for accelerator.

**Comment by Daenyth** [comment]
@poixen We mentioned in #584 that mulligan shouldn't be at the outside edges of the menu in order to reduce risk of accidental activation

**Comment by Daenyth** [comment]
@ideocl4st in case you saw my derp comment about the accelerator key - it's fine, I like the consistency with M being move.

**Comment by tooomm** [comment]
I like the separators @poixen mentioned and @Daenyth objection:

_________
xxx
xxx
-separator-
Open deck in deck editor
_________


_________
xxx
xxx
-separator-
Take Mulligan
-separator-
xxx
_________

**Comment by ideocl4st** [comment]
That makes three of us.

newnewnewhandmenu
newnewlibrarymenu

**Comment by MassCraziness** [comment]
Separators are definitely the way to go. Looks great

**Pull request merged**